omnibus/.kitchen.yml

```diff
@@ -93,7 +94,9 @@ platforms:
       # gets cranky if the mounted Chef source folder is a symlink. This
       # mounts at `C:\vagrant\code` and the Chef source folder is available
       # at `C:\vagrant\code\chef`
-      - ['../..', '/vagrant/code']
+      - ['..', '/vagrant/code/chef']
```

Comment:
This change causes the following error:

```
                          I | 2016-06-21T07:54:07-07:00 | fatal: Invalid symlink
 'C:/vagrant/code/chef': Function not implemented
The following shell command exited with status 128:

    $ git ls-remote "http://git.savannah.gnu.org/r/config.git" "master*"

Output:

    (nothing)

Error:

    fatal: Invalid symlink 'C:/vagrant/code/chef': Function not implemented

C:/languages/ruby/2.1.8/lib/ruby/gems/2.1.0/bundler/gems/omnibus-56b8e507df41/li
b/omnibus/util.rb:101:in `rescue in shellout!'
```

Please change it back to `- ['../..', '/vagrant/code']`. I'll let @ksubrama and @jkeiser give more of the why.